spec/models/druid_version_zip_part_spec.rb

```diff
@@ -51,5 +51,19 @@
         expect(part.hex_to_base64('d41d8cd98f00b204e9800998ecf8427e')).to eq '1B2M2Y8AsgTpgAmY7PhCfg=='
       end
     end
+
+    describe '#md5' do
+      it 'returns MD5 value' do
+        expect(part.md5.to_s).to eq '4f98f59e877ecb84ff75ef0fab45bac5'
+        expect(part.md5).to eq part.metadata[:checksum_md5]
```

Comment:
Only question I have here is that you make an explicit `to_s` cast to make a string comparison in 57, but then you don't cast it in 58 when also comparing to a string.  Probably clearer to say `returns Digest::MD5 object` than *value*.

Reply:
Thanks Joe, I made the changes you suggested and currently pushing changes up.